pkg/kubectl/describe.go

```diff
@@ -895,6 +900,9 @@ func describeContainers(label string, containers []api.Container, containerStatu
 					valueFrom = resolverFn(e)
 				}
 				fmt.Fprintf(out, "      %s:\t%s (%s:%s)\n", e.Name, valueFrom, e.ValueFrom.FieldRef.APIVersion, e.ValueFrom.FieldRef.FieldPath)
+			/*case e.ValueFrom.ResourceFieldRef != nil:
```

Comment:
Nice catch. I forgot I will fix it, i think it requires to pass pod to the function.